app/raid.js

```diff
@@ -789,7 +795,7 @@ class Raid extends Party {
                 .then(message => message.delete({timeout: settings.messageCleanupDelayStatus}))
                 .then(async result => {
                   this.messages.splice(this.messages.indexOf(currentAnnouncementMessage), 1);
-                  await this.persist();
+                  await this.psersist();
```

Comment:
Typo here

Reply:
Well, this is why one doesn't code with a dog laying next to him and on a couch. lol.